Die Zeit.js

```diff
@@ -130,9 +130,11 @@ function scrape(doc, url){
 			keywords[i].trim()
 		)
 	}
-	
+
+	var snapshotNode = ZU.xpath(doc, '//li[@class="article-pager__all"]/a');
+	var snapshotUrl = (snapshotNode.length > 0) ? snapshotNode[0].href : url;
```

Comment:
The correct parameter to show the whole article in a single page will be in the url from the node of class `article-pager__all` (label "Auf einer Seite lesen"), e.g. http://www.zeit.de/2016/06/fluechtlingskrise-grenzen-zuwanderung-loesung-kontigente . For single page article this node does not appear at, e.g. http://www.zeit.de/gesellschaft/zeitgeschehen/2015-06/integration-migration-studie-mipex